src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -115,18 +122,27 @@ foam.CLASS({
     var self = this;
 
       this.addClass(this.myClass())
-      .add(this.slot(function(data, config, config$browseBorder, config$browseViews, config$browseTitle) {
+      .add(this.slot(function(data, config, config$browseBorder, config$browseViews, config$browseTitle, config$browseSubtitle) {
         return self.E()
           .start(self.Rows)
             .addClass(self.myClass('container'))
-            .start(self.Cols)
-              .addClass(self.myClass('header-container'))
-              .start()
-                .addClass(self.myClass('browse-title'))
-                .add(config$browseTitle)
+              .start(self.Rows)
+                .addClass(self.myClass('header-container'))
+                .start(self.Cols)
+                  .start()
+                    .addClass(self.myClass('browse-title'))
+                    .add(config$browseTitle)
+                  .end()
+                  .startContext({ data: self }).tag(self.CREATE).endContext()
+                .end()
+                .callIf(config$browseTitle.length > 0, function() {
```

Comment:
this should be checking browserSubtitle, not browserTitle.

Reply:
Whoops just pushed fix.